### PR TITLE
SERVER add a geoip-downloader dockerfile

### PR DESCRIPTION
**Pull request by fuzzyami**

### CODE REVIEW

**Comment by doodyparizada** [comment]
@fuzzyami If I do [this](https://github.com/rounds/server/blob/master/Dockerfile#L33) in the development docker, will it be terrible?

File: geoip-downloader/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM gliderlabs/alpine:3.1
+
+VOLUME /usr/share/GeoIP
+RUN apk --update add geoip
```
Comment (oryband) [inline geoip-downloader/Dockerfile]: 1. use official alpine, not gilderlabs.
 1. `apk add -qU --no-cache`

File: geoip-downloader/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+ADD databases /usr/share/GeoIP
+ADD databases/GeoIP.conf /etc/
```
Comment (oryband) [inline geoip-downloader/Dockerfile]: stop using ADD, use COPY.

Comment (oryband) [inline geoip-downloader/Dockerfile]: ADD is used for urls

File: geoip-downloader/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+ENTRYPOINT ["geoiplookup"]
```
Comment (oryband) [inline geoip-downloader/Dockerfile]: is `geopiplookup` in the `$PATH`?

**Comment by oryband** [comment]
why is there a binary in the commit?

**Comment by oryband** [comment]
the zip file

**Pull request merged**